js/README.md

```diff
@@ -32,626 +32,66 @@
 
 # JavaScript Guide
 
-Clean, consistent and understandable CSS is paramount to a successful project. It sets the tone, both visually and architecturally, for the entire front-end. These are our collected (and evolving) best practices for writing styles with some _style_. 😎
+*A mostly reasonable approach to JavaScript, inspired by the [Airbnb JavaScript Style Guide](https://github.com/airbnb/javascript).*
 
-- [Organization](#organization)
-  - [Directory Structure](#directory-structure)
-  - [Base Styles](#base-styles)
-  - [Components](#components)
-  - [Utilities](#utilities)
-- [Conventions](#conventions)
-  - [General Rules](#general-rules)
-  - [Property Sorting](#property-sorting)
-  - [Selectors](#selectors)
-  - [Class Naming](#class-naming)
-  - [Comments](#comments)
-- [Architecture](#architecture)
-  - [Dryness](#dryness)
-  - [Encapsulation](#encapsulation)
-  - [Composition](#composition)
-  - [Compatibility](#compatibility)
-- [Tools](#tools)
-  - [my](#blah)
-  - [Processors](#processors)
-  - [Formatters](#formatters)
-  - [Validators](#validators)
-  - [Analyzers](#analyzers)
+## Table of Contents
 
-## Organization
+  1. [References](#references)
```

Comment:
Ironically, I didn't know what "references" _referred_ to until I read it. One small suggestion (not a mandate) would be to include in this table of contents some more language, like "References (Variables, Constants, etc.)"

Reply:
Updated! Let me know your thoughts! 😉